calico-cloud/image-assurance/scanners/cli-based-scanner.mdx

```diff
@@ -0,0 +1,167 @@
+---
+description: Scan images using CLI scanner.
+---
+
+# Manually scan using CLI
```

Comment:
For search results, better title would be: Manually scan images using CLI

Reply:
fixed with bed5e89